programs/librapay_api/src/lib.rs

```diff
@@ -18,11 +18,34 @@ use solana_runtime::loader_utils::load_program;
 use solana_sdk::account::KeyedAccount;
 use solana_sdk::client::Client;
 use solana_sdk::instruction::InstructionError;
+use solana_sdk::message::Message;
 use solana_sdk::pubkey::Pubkey;
-use solana_sdk::signature::Keypair;
+use solana_sdk::signature::{Keypair, KeypairUtil};
+use solana_sdk::system_instruction;
 
 use types::account_address::AccountAddress;
 
+pub fn create_genesis<T: Client>(from_key: &Keypair, client: &T, amount: u64) -> Keypair {
+    let libra_genesis_key = Keypair::new();
+
+    let instruction = system_instruction::create_account(
+        &from_key.pubkey(),
+        &libra_genesis_key.pubkey(),
+        1,
+        2048 as u64,
```

Comment:
Does it need a cast?

Reply:
Switched it to actually calculate the size needed, which then does need the cast :-)